support/client/lib/vwf/view/blockly.js

```diff
@@ -251,6 +251,12 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
                             // If the new active node is different than the old,
                             // then we need to load its program into the toolbox
                             if ( show ) {
+                                if ( Blockly.mainWorkspace ) {
+                                    Blockly.mainWorkspace.maxBlocks = blocklyNode.ramMax;
```

Comment:
This is an application specific value (`ramMax`). This should not be a driver change.